addons/website/models/website.py

```diff
@@ -1152,9 +1152,11 @@ def create(self, vals):
     @api.multi
     def unlink(self):
         default_menu = self.env.ref('website.main_menu', raise_if_not_found=False)
+        menus_to_remove = self
         for menu in self.filtered(lambda m: default_menu and m.parent_id.id == default_menu.id):
-            self.env['website.menu'].search([('url', '=', menu.url), ('id', '!=', menu.id)]).unlink()
-        return super(Menu, self).unlink()
+            menus_to_remove = menus_to_remove | \
```

Comment:
ew a backslash

Reply:
do you prefer 150+ chars lines? or parenthesis?

Reply:
Parentheses are the most [pythonic](https://www.python.org/dev/peps/pep-0008/#maximum-line-length) but here you could just use `|=` :trollface: